wodles/aws/tests/test_aws.py

```diff
@@ -90,6 +90,53 @@ def test_metadata_version_services(mocked_db, class_):
         assert(metadata_version == ins.wazuh_version)
 
 
+@patch('os.path.exists', return_value=False)
+@patch('aws_s3.botocore.config.Config')
+@patch('aws_s3.debug')
+def test_default_config_no_aws_config(mock_debug, mock_config, mock_exists):
+    """
+    Test the default configuration generation when no AWS config file is present.
+    
+    Parameters:
+        mock_debug (MagicMock): Mock object for the 'aws_s3.debug' function.
+        mock_config (MagicMock): Mock object for the 'aws_s3.botocore.config.Config' class.
+        mock_exists (MagicMock): Mock object for the 'os.path.exists' function.
```

Comment:
According to the format we use this should be
```suggestion
    Parameters:
        mock_debug : MagicMock
            Mock object for the 'aws_s3.debug' function.
        mock_config : MagicMock
            Mock object for the 'aws_s3.botocore.config.Config' class.
        mock_exists : MagicMock
            Mock object for the 'os.path.exists' function.
```

However, since this is a test, you may omit explaining the parameters altogether.